packages/wrangler/src/__tests__/d1/convert-timestamp-to-iso.test.ts

```diff
@@ -3,7 +3,7 @@ import { convertTimestampToISO } from "../../d1/timeTravel/utils";
 
 describe("convertTimestampToISO", () => {
 	beforeAll(() => {
-		vi.useFakeTimers();
+		vi.useFakeTimers({ toFake: ["setTimeout", "clearTimeout", "Date"] });
```

Comment:
Why do we need to specify these? It feels like we are coupling the test with the implementation if we have to specify the methods to fake.

Reply:
This restores the Vitest 2 defaults: https://vitest.dev/guide/migration#fake-timers-defaults. Without this, our tests were hanging (presumably because `performance.now()` was not previously being mocked?